Undefined: FS (counter example won't run)

I just cloned the repo, installed the latest go version `go1.14.7 windows/amd64` and tried to run the counter example but got an error:

```
C:\Users\karl\tst\lorca\examples\counter>go get
go: downloading golang.org/x/net v0.0.0-20200222125558-5a598a2470a0

C:\Users\karl\tst\lorca\examples\counter>go run main.go
# command-line-arguments
.\main.go:68:36: undefined: FS

C:\Users\karl\tst\lorca\examples\counter
```

--- a/README.md
+++ b/README.md
@@ -80,7 +80,7 @@ Here are the steps to run the hello world example.
 ```
 cd examples/counter
 go get
-go run main.go
+go run ./
 ```
 
 ## How it works

